types/multiparty/index.d.ts

```diff
@@ -12,7 +12,14 @@ export declare class Form extends events.EventEmitter {
      * @param request
      * @param callback
      */
-    parse(request: http.IncomingMessage, callback?: (error: Error, fields: any, files: any) => any): void;
+    parse(
+        request: http.IncomingMessage,
+        callback?: (
+            error: Error | null,
+            fields: Record<string, string[]>,
+            files: Record<string, string[]>,
```

Comment:
`Record<string, string[]>` or `Record<string, string[] | undefined>`?

Since this isn't an input position, maybe worthwhile to note that some props may not be present.

Reply:
Done.